server/src/main.rs

```diff
@@ -0,0 +1,114 @@
+//! gRPC server implementation
+
+///module svc_scheduler generated from svc-scheduler.proto
+pub mod svc_scheduler {
+    #![allow(unused_qualifications)]
+    include!("svc_scheduler.rs");
+}
+
+use dotenv::dotenv;
+use std::env;
+use std::time::SystemTime;
+use svc_scheduler::scheduler_server::{Scheduler, SchedulerServer};
+use svc_scheduler::{
+    CancelFlightResponse, ConfirmFlightResponse, FlightPriority, FlightStatus, Id, QueryFlightPlan,
+    QueryFlightRequest, QueryFlightResponse, ReadyRequest, ReadyResponse,
+};
+use tonic::{transport::Server, Request, Response, Status};
+
+///Implementation of gRPC endpoints
+#[derive(Debug, Default, Copy, Clone)]
+pub struct SchedulerImpl {}
+
+#[tonic::async_trait]
+impl Scheduler for SchedulerImpl {
+    async fn query_flight(
+        &self,
+        request: Request<QueryFlightRequest>, // Accept request of type QueryFlightRequest
+    ) -> Result<Response<QueryFlightResponse>, Status> {
+        // Return an instance of type QueryFlightResponse
+        println!("Got a request: {:?}", request);
+
+        let requested_time = request.into_inner().requested_time;
```

Comment:
Minor comment: please add comments for these implementations indicating they're placeholders

Would a comment block above `query_flight` be unusual in gRPC implementation? Would be a nice area to explain what's going on in the function when we eventually contact svc-storage, lib-router, etc.

Same thing for the functions below - indicate in a comment block that they're currently returning an arbitrary value